### PR TITLE
Notebook viewer

### PR DESCRIPTION
**Pull request by jasongrout**
This integrates a notebook viewer plugin.

### CODE REVIEW

**Comment by jasongrout** [comment]
@blink1073 - comments are welcome!

File: example/webpack.conf.js [hunk 1]
```diff
@@ -7,11 +7,12 @@ module.exports = {
   },
   node: {
     fs: "empty"
-  },
+  }, debug: true,
```
Comment (blink1073) [inline example/webpack.conf.js]: Is this intended to be kept?  If so, it should be on its own line.

File: package.json [hunk 2]
```diff
@@ -31,14 +33,14 @@
...
-    "typescript": "^1.6.2",
+    "typedoc": "^0.3.12",
+    "typescript": "^1.7.5",
     "webpack": "^1.12.9"
   },
   "scripts": {
     "clean": "rimraf docs && rimraf lib && rimraf test/build",
     "clean:example": "rimraf example/build",
-    "build:example": "tsc --project example/src && webpack --config example/webpack.conf.js",
+    "build:example": "tsc --project example/src && webpack -d --config example/webpack.conf.js",
```
Comment (blink1073) [inline package.json]: Duplicate debug switch.

**Comment by blink1073** [comment]
Looks ready to merge otherwise, nice!

**Comment by blink1073** [comment]
Oh, did you want the tab title to still have the `.ipynb` extension?  Because it does.

**Comment by jasongrout** [comment]
It made a huge difference in debugging to have the webpack debug switch.  I'll decide on one or the other ways of specifying it :).

**Comment by jasongrout** [comment]
eh, I'll leave the .ipynb there for now.

**Comment by blink1073** [comment]
:tada:

**Pull request merged**